tbtaf/test/discoverer/discoverer_tester.py

```diff
@@ -2,40 +2,42 @@
 Created on 20/11/2015
 '''
 
+from __future__ import absolute_import
```

Comment:
Please add the corresponding unit tests folder and content for the Databridge and Publisher additions

Reply:
@mgonzalez1986

Reply:
implemented, please re check